airbyte-integrations/connectors/source-amazon-seller-partner/integration_tests/configured_catalog_browse_tree_data.json

```
@@ -0,0 +1,136 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "GET_XML_BROWSE_TREE_DATA",
+        "json_schema": {
+          "title": "XML Browse Tree Data",
+          "description": "XML Browse Tree Data",
+          "type": "object",
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "properties": {
+            "browseNodeId": {
+              "type": ["null", "string"]
```

```suggestion
              "type": "string"
```
As an identifier, I'm not sure this field can be null.